src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -62,30 +68,30 @@ function createHederaService() {
     return BigNumber(value).multipliedBy(_precision);
   };
 
-  const createLiquidityPool = async () => {
-    const tokenAQty = withPrecision(200);
-    const tokenBQty = withPrecision(220);
-    console.log(`Creating a pool of ${tokenAQty} units of token A and ${tokenBQty} units of token B.`);
-    const liquidityPool = await new ContractExecuteTransaction()
-      .setContractId(contractId)
-      .setGas(2000000)
-      .setFunction(
-        "initializeContract",
-        new ContractFunctionParameters()
-          .addAddress(treasuryId.toSolidityAddress())
-          .addAddress(tokenA)
-          .addAddress(tokenB)
-          .addInt64(tokenAQty)
-          .addInt64(tokenBQty)
-          .addInt256(new BigNumber(10)) //fee
-      )
-      .freezeWith(client)
-      .sign(treasuryKey);
-    const liquidityPoolTx = await liquidityPool.execute(client);
-    const transferTokenRx = await liquidityPoolTx.getReceipt(client);
-    console.log(`Liquidity pool created: ${transferTokenRx.status}`);
-    await pairCurrentPosition(contractId);
-  };
+  // const createLiquidityPool = async (contractId: ContractId) => {
```

Comment:
Any reason we're commenting out this block instead of removing it?

Reply:
I searched this function it was not being used anywhere so I was not sure whether do we need it or remove it, for now I have removed it completely.